src/include/daos_uns.h

```diff
@@ -121,6 +121,8 @@ struct duns_attr_t {
 /** Length of the extended attribute */
 #define DUNS_MAX_XATTR_LEN	170
 
+#define DUNS_XATTR_FMT		"DAOS.%s://%36s/%36s"
```

Comment:
I know I may be the guilty original guy, but I agree with the oxygen advice that some detailing comment is a good idea !

Reply:
Oh yeah, good point.  We want Doxygen to be clean for 2.0 so the plan is to resolve all of these shortly and disable landing patches which have issues.